app/views/competitions/fixtures/index.html.erb

```diff
@@ -1,41 +1,37 @@
 <h1><%= @competition.name %></h1>
-<% if @competition.stages.count == 1 %>
-    <ul>
-        <% @competition.stages.first.matchdays.order(:order).each do |fixture| %>
-            <li>
-                <a href="/fixtures/<%= fixture.id %>">
-                    <p><%= fixture.name %></p>
-                </a>
-            </li>
-        <% end %>
-    </ul>
-<% else %>
-    <ul>
-        <% @competition.stages.each do |stage| %>
-            <li>
-                <p><%= stage.name %></p>
-                <% if stage.is_a?(LeagueStage) %>
-                    <ul>
-                        <% stage.matchdays.each do |matchday| %>
-                            <li>
-                                <a href="/fixtures/<%= matchday.id%>">
-                                    <p><%= matchday.name %></p>
-                                </a>
-                            </li>
-                        <% end %>
-                    </ul>
-                <% else %>
-                    <ul>
-                        <% stage.rounds.each do |round| %>
-                            <li>
-                                <a href="/fixtures/<%= round.id %>">
-                                    <p><%= round.name %>
-                                </a>
-                            </li>
-                        <% end %>
-                    </ul>
-                <% end %>
-            </li>
-        <% end %>
-    </ul>
-<% end %>
+<div>
+  <%= link_to 'Home', home_path %>
+  <%= ' > ' %>
+  <%= link_to I18n.t('activerecord.models.competition_series'), competition_series_list_path %>
+  <%= ' > ' %>
+  <%= link_to @competition.series.name, competition_series_path(@competition.series) %>
+  <%= ' > ' %>
+  <%= link_to @competition.name, competition_path(@competition) %>
+  <%= " > #{I18n.t('pages.fixtures.title')}" %>
+<div>
+<ul>
+    <% @competition.stages.each do |stage| %>
+        <li>
+            <% if @competition.stages.count > 1%>
+                <%= stage.name %>
+            <% end %>
+            <% if stage.is_a?(LeagueStage) %>
+                <ul>
+                    <% stage.matchdays.each do |matchday| %>
+                        <li>
+                            <%= link_to matchday.name, fixture_path(matchday) %>
```

Comment:
[bug] Fixture name is not displayed when the DB value is null

Reply:
TODO: Add `name` method to `Matchday` model, which uses I18n translation to automatically calculate the name instead of using the DB value